utils/User.ts

```diff
@@ -424,7 +417,7 @@ export class User {
   }
 
   async updateL1Asset(
-    assetId: BN,
+    assetId: any,
```

Comment:
you can use extended type instead, like
assetId: BN | null ,
I think that we should keep types there instead of using "any"

Reply:
All works well with `BN` format. Done